tika-parsers/pom.xml

```diff
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.tika</groupId>
     <artifactId>tika-parent</artifactId>
-    <version>1.13-SNAPSHOT</version>
+    <version>1.14-SNAPSHOT</version>
```

Comment:
Actually I did not make these changes. It was already present when I cloned the repository Professor.